akka-remote-tests/src/multi-jvm/scala/akka/remote/artery/AeronStreamLatencySpec.scala

```diff
@@ -80,6 +82,9 @@ abstract class AeronStreamLatencySpec
 
   val driver = MediaDriver.launchEmbedded()
 
+  val stats =
+    new AeronStat(AeronStat.mapCounters(new File(driver.aeronDirectoryName, CncFileDescriptor.CNC_FILE)))
```

Comment:
"command and control", see description of its layout in io.aeron.CncFileDescriptor